Supertext.Base/Net/Mail/EmailInfoTemplates.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+
+namespace Supertext.Base.Net.Mail
+{
+    public class EmailInfoTemplates
+    {
+        public EmailInfoTemplates(string templateId, string dynamicTemplateDataAsJson, PersonInfo from, PersonInfo to,
+                                  string subject)
```

Comment:
Why is only parameter on its own line? Either they should all be on the same line, or they should all be their own lines.

Reply:
fixed